docs/CONTRIBUTE.md

```diff
@@ -20,9 +20,10 @@ Development takes place on GitHub, where the git-flow branch structure is used:
 
 ### Prepare development environment
 It is recommended to use Unix systems. You can use free Oracle Virtualbox and Ubuntu.  
-Setup script for Ubuntu system (python version might be 3.6+):
+Setup script for Ubuntu system (python version might be 3.7+):
```

Comment:
I realize it was like this in the original but if we're changing the line, as well update

```suggestion
Setup script for Ubuntu system (python version must be 3.7+):
```